cockatrice/src/tab_game.cpp

```diff
@@ -1192,6 +1193,10 @@ void TabGame::eventJoin(const Event_Join &event, int /*eventPlayerId*/, const Ga
     } else {
         Player *newPlayer = addPlayer(playerId, playerInfo.user_info());
         messageLog->logJoin(newPlayer);
+        if (trayIcon) {
+            trayIcon->showMessage(tr("A player has joined your game"),
```

Comment:
I like the idea! Maybe say "A player joined game #12345" and then the body as "PLAYER has joined the game" because it might not be "your" game per say. Overall, awesome idea :)

Reply:
@ZeldaZach good point!  I made the change.